frontend/src/host_orchestrator/orchestrator/cvd/cvd.go

```diff
@@ -122,6 +124,50 @@ func (c *Command) Run() error {
 	return nil
 }
 
+// Same as `Run()`, but error won't be populated in case of failed executions, the caller gets the exit code value.
```

Comment:
Can `Run` be implemented by calling `Execute` then and processing the exit code it returns?

Reply:
simplified the PR so it executes cvd stop only, this is no longer relevant.